### PR TITLE
Enable active standby for coordinators by default

### LINKED ISSUE #25956: [Enhancement]: Enable active standby by default
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What would you like to be added?

Enable active standby by default

### Why is this needed?

Milvus support HA via active standby but this feature is disable by default which caused some trouble for users.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by congqixia**
Resolves #25956
/kind improvement

### CODE REVIEW

File: configs/milvus.yaml
Comment (elstic) [inline configs/milvus.yaml:1]: We also mentioned the issue that we want to set it to true: https://github.com/milvus-io/milvus/issues/25957

But does indexCoord also add it and set it to true?

Comment (congqixia) [reply configs/milvus.yaml:1]: @elstic in 2.3 indexcoord is merged into datacoord and shares the same config value here

Comment (elstic) [reply configs/milvus.yaml:1]: > @elstic in 2.3 indexcoord is merged into datacoord and shares the same config value here

okay, I have no questions.

**Comment by congqixia** [comment]
enabling `activeStandy` by default in configuration will cause integration test broken for now. will fix this problem before merging this pr.
/hold

**Pull request closed without merging**